vendor/engines/split_accounts/README.md

```diff
@@ -0,0 +1,64 @@
+# Split Accounts
+
+Splits Accounts is an engine that allows a user to charge a certain percentage
+of a purchase to different accounts.
+
+## Installation
+
+Make sure the `gem "split_accounts",   "~> 0.0.1", path: "vendor/engines/split_accounts"`
+line is enabled in `Gemfile`. Then ensure the `feature.split_accounts_on` in
+`Settings.yml` is set to `true`.
+
+## Notes
+
+* Only global admins may create a Split Account. Other account admins and facility
+    admins may edit the membership list.
+* `SplitAccounts` expire when the earliest of its children expire
+* If you suspend a child account, the parent gets suspended as well. However, if you
+    unsuspend a child account, the parent remains suspended. You must re-activate
+    the parent account yourself.
+
+## "Apply Remainder" Behavior
+
+The costs are calculated by multiplying the cost of an order by the percentage of
+each split. The result is then rounded down to the penny. We then add up each
+accounts' amount; if there is a difference with the original amount, the difference
+is applied to the account that has "Apply Remainder" selected.
```

Comment:
I've completely renamed "extra penny" "apply remainder" even down at the database level.